retrying.py

```diff
@@ -197,6 +222,21 @@ def exponential_sleep(self, previous_attempt_number, delay_since_first_attempt_m
             result = 0
         return result
 
+    def exponential_sleep_with_decorrelated_jitter(self, previous_attempt_number, delay_since_first_attempt_ms):
```

Comment:
@charlescbeebe would be good if instead of a commented block here it was a [docstring](https://www.python.org/dev/peps/pep-0257/) enclosed in `"""`s.